src/index.tsx

```
@@ -9,9 +9,9 @@
   CONFIG_FILE,
   CONFIG_DIR,
 } from './utils/types.js';
-import { startupCheck, readConfigFile } from './utils/config.js';
+import { startupCheck, readConfigFile, backupClijs, backupNativeBinary, restoreClijsFromBackup, restoreNativeBinaryFromBackup } from './utils/config.js';
```

ESLint

[error] 'restoreClijsFromBackup' is defined but never used.

(@typescript-eslint/no-unused-vars)